### PR TITLE
Adding suggest and accuse btn and adding styling to modals

### PR DESCRIPTION
**Pull request by michellegaletti**
accuseSuggestModal
mobileModal
accuseWebModal
accuseMobileModal
winLoseWebModal
loseWinMobileModal

### CODE REVIEW

**Review by michaelgraves8 — APPROVED** [review]
looks great!

File: src/sass/_mixins.scss [hunk 1]
```diff
@@ -39,3 +39,74 @@
...
+  margin-bottom: 5%;
+  padding-left: 5%;
+  padding-right: 5%;
+  text-align: center;
+  font-family: $font-main;
+}
+
+@mixin selectedTextStyle {
+  font-weight: 600;
+  color: $color-dark;
```
Comment (asalduur) [inline src/sass/_mixins.scss]: can we actually do `color: $color-darkest;` here? just to increase the contrast between the background and test.

**Review by asalduur — CHANGES_REQUESTED** [review]
on the fourth image, for the bold words, can we actually do `color: $color-darkest;` ? just to increase the contrast between the background and text.

File: src/sass/_mixins.scss (hunk 1 above)
Comment (michellegaletti) [reply src/sass/_mixins.scss]: I tested it out and it is too similar to the black, I can create a different dark blue color, would that be better?

Comment (asalduur) [reply src/sass/_mixins.scss]: i see, if you can find an in between shade, that works too!

**Review by asalduur — APPROVED** [review]

**Pull request merged**